src/openms/include/OpenMS/DATASTRUCTURES/DataValue.h

```diff
@@ -116,7 +116,7 @@ namespace OpenMS
     /// copy constructor
     DataValue(const DataValue&);
     /// destructor
-    virtual ~DataValue();
+    ~DataValue();
```

Comment:
I dont think that people should use DataValue polymorphically or even derive directly from it at all.